spec/gis_robot_suite/raster_normalizer_spec.rb

```diff
@@ -116,11 +116,6 @@
           "gdal_translate -expand rgb /tmp/normalizeraster_bb021mm7809/raw8bit.tif /tmp/normalizeraster_bb021mm7809/MCE_FI2G_2014.tif -co 'COMPRESS=LZW'",
           exception: true
         )
-        # Adds an alpha channel
-        expect(Kernel).to have_received(:system).with(
-          'gdalwarp -dstalpha /tmp/normalizeraster_bb021mm7809/MCE_FI2G_2014.tif /tmp/normalizeraster_bb021mm7809/MCE_FI2G_2014_alpha.tif',
-          exception: true
-        )
```

Comment:
at first i'd changed this to:
```ruby
        # Does not add an alpha channel
        expect(Kernel).not_to have_received(:system).with(
          /gdalwarp -dstalpha/,
          exception: true
        )
```

but that felt a little brittle and overly specific (maybe there are other commands for adding alpha channel, flag doesn't necessarily have to follow the command name directly, etc).  easy to add back by dropping the second commit.